tests/unit/s2n_session_ticket_test.c

```diff
@@ -1348,6 +1348,50 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_config_free(config));
     }
 
+    /* Test: TLS1.3 resumption is successful when key used to encrypt ticket is in decrypt-only state */
+    if (s2n_is_tls13_fully_supported()) {
+        DEFER_CLEANUP(struct s2n_connection *client_connection = s2n_connection_new(S2N_CLIENT), s2n_connection_ptr_free);
+
+        /* Set client ST and session state */
+        EXPECT_SUCCESS(s2n_connection_set_session(client_connection, tls13_serialized_session_state.blob.data,
+                s2n_stuffer_data_available(&tls13_serialized_session_state)));
+
+        DEFER_CLEANUP(struct s2n_config *client_configuration = s2n_config_new(), s2n_config_ptr_free);
```

Comment:
Can you nullcheck the config object that you create? Like EXPECT_NOT_NULL(config)? Also nullcheck the connection objects that you create?